astro/src/content/docs/get-started/core-concepts/collected-metrics.mdx

```diff
@@ -0,0 +1,535 @@
+    "webhookUserLoginSuccessTransactional": 5,
+    "webhookUserLoginSuspicious": 5,
+    "webhookUserLoginSuspiciousTransactional": 5,
+    "webhookUserPasswordBreach": 4,
+    "webhookUserPasswordBreachTransactional": 6,
+    "webhookUserPasswordResetSend": 4,
+    "webhookUserPasswordResetStart": 4,
+    "webhookUserPasswordResetSuccess": 4,
+    "webhookUserPasswordUpdate": 4,
+    "webhookUserReactivate": 5,
+    "webhookUserReactivateTransactional": 6,
+    "webhookUserRegistrationCreate": 4,
+    "webhookUserRegistrationCreateComplete": 4,
+    "webhookUserRegistrationCreateTransactional": 6,
+    "webhookUserRegistrationDelete": 4,
+    "webhookUserRegistrationDeleteComplete": 5,
+    "webhookUserRegistrationDeleteTransactional": 6,
+    "webhookUserRegistrationUpdate": 4,
+    "webhookUserRegistrationUpdateComplete": 4,
+    "webhookUserRegistrationUpdateTransactional": 6,
+    "webhookUserRegistrationVerified": 4,
+    "webhookUserRegistrationVerifiedTransactional": 6,
+    "webhookUserTwoFactorMethodAdd": 3,
+    "webhookUserTwoFactorMethodRemove": 5,
+    "webhookUserUpdate": 4,
+    "webhookUserUpdateComplete": 4,
+    "webhookUserUpdateTransactional": 6
+  }
+}
+```
+</details>
+
+You can see here that we are collecting information about feature usage.
+
+We will never collect anything personally identifiable. We may collect new metrics as we add new features, but those will be outlined in our release notes.
+
+Here's the breakdown of who we will collect data from, and how it will happen:
+
+-   Self-hosted: You get to choose whether or not you want to share your data with us. You can turn usage data collection off by API or in the admin UI. You can also block the outbound request at the network--just block the hostname https://usage-stats.fusionauth.io. If you decide to share usage data, know that we'll handle your data with the highest level of care and security.
+-   By default, any upgraded instances will NOT send usage data.
+-   By default, any new instances will send usage data, but you can opt out at instance creation or any time after.
+-   FA-Cloud: For those who choose to have us host your FusionAuth deployment, data collection is part of the package. It helps us give you top-notch service and security, and we promise that we're only using that data to make FusionAuth better for you.
+-   Community plan: If you're using an unlicensed version of FusionAuth, we'll only be collecting data points. If you do choose Licensed Community (available in version 1.52.0 and later, which adds free passkeys and other features), we will collect the same data, but tie it to your account and identity.
+-   Other plans: We collect the same data and tie it to your account and identity.
+
+If you have questions or concerns, we're here to chat. Drop us an email at [devrel@fusionauth.io](mailto:devrel@fusionauth.io), or stop by the [FusionAuth Community forums](/community).
+
+FAQ: 
+-----
+
+### How will this impact performance? 
```

Comment:
Is this the proper casing?

Reply:
Well, that depends. If we're using the Inversoft style guide then yes. Though if I'm not mistaken, don't we have an automation that changes everything to the correct case anyway?

Reply:
How does it render if you spin up the local site?